db/migrate/20240108141151_rename_starts_on_column_in_schedules.rb

```diff
@@ -0,0 +1,5 @@
+class RenameStartsOnColumnInSchedules < ActiveRecord::Migration[7.1]
```

Comment:
maybe we can move this into the initial migration to avoid changes as this is a new table? (understand this might be hard for you locally to revert etc.)